README.md

```diff
@@ -111,6 +113,8 @@ Now, on our servers, we can set the environment variable appropriately. On
 our local development machines, exceptions will never be sent, because the
 default value is not in the list of `included_environments`.
 
+Sentry uses the [hackney HTTP client](https://github.com/benoitc/hackney) for HTTP requests.  If you need to set hackney configurations for things like a proxy or different pool, the `hackney_opts` configuration is passed directly to hackney.
```

Comment:
I'd deep link to the options https://github.com/benoitc/hackney/blob/master/doc/hackney.md#request5 (not easy to find)